ext/session/tests/user_session_module/skipif.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+$save_path = ini_get("session.save_path");
+if ($save_path) {
+    if (!file_exists($save_path)) {
+        die("skip Session save_path doesn't exist");
+    }
+
+    if ($save_path && !@is_writable($save_path)) {
+        if (($p = strpos($save_path, ';')) !== false) {
+            $save_path = substr($save_path, ++$p);
+        }
+        if (!@is_writable($save_path)) {
+            die("skip session.save_path $save_path is not writable\n");
+        }
+    }
+}
+?>
```

Comment:
Yes, it is a duplicate, but I *thought* I could get rid of the skipif section all together and not rely on the file system but that doesn't seem likely so the easy fix was to just copy the file again so that I didn't need to amend all paths. But I can try again to get rid or point to the root SKIPIF file.

Reply:
Turns out it's completely useless, even for the normal session tests, as it's meant to check if the INI save path exists and is writeable, but this is never set in our tests.